service/history/workflow/context.go

```diff
@@ -869,9 +877,23 @@ func (c *ContextImpl) UpdateRegistry(ctx context.Context) update.Registry {
 }
 
 // Returns true if execution is forced terminated
-func (c *ContextImpl) enforceSizeCheck(
+func (c *ContextImpl) enforceHistorySizeCheck(
 	ctx context.Context,
 ) (bool, error) {
+	// Hard terminate workflow if still running and breached history size, history count, or ms size limits
```

Comment:
confusing that name says this enforces only history size, but comments says ms size enforced here too